cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -11,36 +11,97 @@
 import io.strimzi.api.kafka.model.storage.PersistentClaimStorage;
 import io.strimzi.api.kafka.model.storage.SingleVolumeStorage;
 import io.strimzi.api.kafka.model.storage.Storage;
+import io.strimzi.operator.cluster.model.AbstractModel;
+import io.strimzi.operator.cluster.model.VolumeUtils;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
 
 import java.util.List;
 
 import static io.strimzi.operator.cluster.model.StorageUtils.parseMemory;
 
 public class Capacity {
-    public static final double DEFAULT_BROKER_DISK_MIB_CAPACITY = 100_000;  // in MiB
-    public static final int DEFAULT_BROKER_CPU_UTILIZATION_CAPACITY = 100;  // as a percentage (0-100)
-    public static final double DEFAULT_BROKER_INBOUND_NETWORK_KIB_PER_SECOND_CAPACITY = 10_000;  // in KiB/s
-    public static final double DEFAULT_BROKER_OUTBOUND_NETWORK_KIB_PER_SECOND_CAPACITY = 10_000;  // in KiB/s
+    private static final int DEFAULT_BROKER_ID = -1;
```

Comment:
When using a non-JBOD disk configuration, we can generate a generic capacity configuration that contains a single broker entry, the "default" broker configuration, which will apply to all brokers. CC designates the id of this default broker entry as  `-1`

Reply:
Ok, makes sense. Can you add some explanatory comment? It will make the code more readable and understandable even for people who are not CC experts. ;-)